ansible/group_vars/catalog-next/vars.yml

```diff
@@ -39,3 +41,4 @@ catalog_ckan_plugins_default:
   - dcat
   - dcat_json_interface
   - structured_data
+  - saml2
```

Comment:
This will apply to all environments, like staging and production where you have `catalog_next_ckan_saml2_enabled: false`.

Instead, I would set `catalog_ckan_plugins_additional` in the environments where you're enabling saml2 and leave this list alone.

Reply:
Good idea!  Resolved.